src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs_tab.cljs

```diff
@@ -54,7 +57,32 @@
       (let [server-response (:server-response @state)
             {:keys [configs error-message]} server-response]
         (cond
-          (nil? server-response) [comps/Spinner {:text "Loading configurations..."}]
+          (nil? server-response) [comps/Spinner {:text
```

Comment:
Bring the `[comps/Spinner...` down to the next line so all of this doesn't get smushed over